src/template/partials/header.html

```diff
@@ -0,0 +1,54 @@
+<header>
+   
+                    
+        <nav class="navbar navbar-expand-lg bg-body-tertiary">
+            <div class="container">
+                
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon">
+                        <img src="assets/images/bar.svg" alt="bar">
+                    </span>
+                </button>
+
+                <h1>
+                <a class="navbar-brand" href="#header-sec">
```

Comment:
The logo anchor should redirect to homepage

Reply:
done

Reply:
#home-page !!!!!!